dev-libs/libratbag/libratbag-0.13.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit meson python-r1 udev
+
+DESCRIPTION="Library to configure gaming mice"
+HOMEPAGE="https://github.com/libratbag/libratbag"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc elogind systemd test"
+REQUIRED_USE="^^ ( systemd elogind ) ${PYTHON_REQUIRED_USE}"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	${PYTHON_DEPS}
+	virtual/pkgconfig
+	doc? ( app-doc/doxygen )
+	elogind? ( sys-auth/elogind )
+	systemd? ( sys-apps/systemd )
+	dev-python/python-evdev[${PYTHON_USEDEP}]
+	dev-lang/swig
+	test? ( dev-util/valgrind )
+"
+RDEPEND="
+	${PYTHON_DEPS}
+	dev-libs/libevdev
+	virtual/libudev
+"
+
+src_prepare() {
+	default
+
+	sed '/default:/d' -i meson_options.txt || die
```

Comment:
Add a comment above this sed to explain what it does.

Reply:
If this did something in the past, it doesn't do anything now.  removed.